_posts/2020-10-15-september-rubygems-update.md

```diff
@@ -0,0 +1,67 @@
+---
+title: September 2020 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+---
+
+# September 2020 RubyGems Updates
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems, Ruby Toolbox and RubyGems.org in September.
+
+## RubyGems News
+This month, We did a lot of work triaging issues and so far we're "**winning the pulse**" with over **80 issues**  closed in relation to only **14 opened**, and **44 PRs** merged in relation to **8 opened**. We've also fixed a couple of new and outstanding issues some of which include:
+
+* [Fixed Resolver recently generating duplicate spec groups](https://github.com/rubygems/rubygems/pull/3965) making debug output much more verbose than it should.
+* Installed a [local `gemspec` without dependencies unnecessarily hitting the network](https://github.com/rubygems/rubygems/pull/3968).
```

Comment:
I'd go with

```suggestion
* Fixed installing a local `gemspec` without dependencies [unnecessarily hitting the network](https://github.com/rubygems/rubygems/pull/3968).
```

since it explains the problem better and adds the link around the actual issue.